durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts

```diff
@@ -171,48 +171,29 @@ export class TaskHubsCollector {
     }
 
     // Creates a user-specific access token for accessing Storage, also adds other needed headers
-    private async CreateIdentityBasedAuthHeadersForTableStorage(tokenCredential: any): Promise<{}> {
+    private async CreateIdentityBasedAuthHeadersForTableStorage(azureSubscription: AzureSubscription): Promise<{}> {
 
-        let token = '';
+        // Looks like azureSubscription.credential.getToken(scopes) does not respect scopes (getting a token with default 'https://management.core.windows.net' audience instead of Storage-specific scope) 
+        // So will have to use vscode.authentication directly
```

Comment:
interesting - do you know if this is a known bug somewhere?

Reply:
Turns out it is: https://github.com/microsoft/vscode-azuretools/issues/1596

Reply:
Makes sense. My main suggestion would be to link the GitHub issue there in the comments, but it's non-blocking

Reply:
Done. Thanks!